app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    def die(self) -> None:
+        if self.health <= 0:
+            Animal.alive.remove(self)
```

Comment:
The `die` method checks if `self.health` is less than or equal to zero to remove the animal from the `alive` list. Ensure that this method is called whenever an animal's health is reduced to zero or below, such as after a `bite`.